src/toolkits/drawing_classifier/dc_data_iterator.hpp

```diff
@@ -0,0 +1,177 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#ifndef TURI_DRAWING_CLASSIFICATION_DC_DATA_ITERATOR_HPP_
+#define TURI_DRAWING_CLASSIFICATION_DC_DATA_ITERATOR_HPP_
+
+#include <random>
+#include <string>
+#include <vector>
+
+#include <core/data/sframe/gl_sframe.hpp>
+#include <ml/neural_net/float_array.hpp>
+
+const int kDrawingHeight = 28;
+const int kDrawingWidth = 28;
+const int kDrawingChannels = 1;
```

Comment:
These should
* Be moved inside the `drawing_classifier` namespace
* Use `constexpr` instead of `const`, since they are compile-time constants, not run-time computed (https://google.github.io/styleguide/cppguide.html#Use_of_constexpr)

Reply:
Done, thanks!